README.md

```diff
@@ -66,9 +66,50 @@ Build instructions:
 	make
 
 This assumes you make your build dir a subdirectory of the ceph.git
-checkout.  If you put it elsewhere, just replace .. above with a
+checkout. If you put it elsewhere, just replace .. above with a
 correct path to the checkout.
 
+To build only certain targets use:
+
+        make [target name]
+
+To install:
+
+        make install
+ 
+CMake Options
+-------------
+
+If you run the `cmake` command by hand, there are many options you can
+set with "-D". For example the option to build the RADOS Gateway is
+defaulted to ON. To build without the Rados Gateway:
+
+        cmake -DWITH_RADOSGW=OFF [path to top level ceph directory]
+
+Another example below is building with debugging and alternate locations 
+for a couple of external dependencies:
+
+        cmake -DLEVELDB_PREFIX="/opt/hyperleveldb" -DOFED_PREFIX="/opt/ofed" \
+        -DCMAKE_INSTALL_PREFIX=/opt/accelio -DCMAKE_C_FLAGS="-O0 -g3 -gdwarf-4" \
+        ..
+
+To view an exhaustive list of -D options, you can invoke `cmake` with:
+
+        cmake -LH
```

Comment:
@ivancich actually cmake -LH does do the trick from what I found. What do you think?